src/app/landing/LandingPage.jsx

```diff
@@ -57,34 +64,90 @@ class LandingPage extends Component {
               <h3>How it works</h3>
               <h2>Search for what you need</h2>
               <p className="text-lighter">
-                GoGetta has New York’s most up-to-date service information on food, personal care, shelter, and clothing.
+                GoGetta has New York’s most up-to-date service information on
+                food, personal care, shelter, and clothing.
               </p>
             </div>
 
             <picture>
+              {/* desktop jpg */}
```

Comment:
As I'm sure I've noted in the past, I always consider comments a red flag that something in the code isn't as readable as it could be.

In this case, one easy improvement, that would not only make this clear but also save you duplication and thus make this less error-prone, would be to put `1224` into a variable with an indicative name (e.g. desktopPictureMinWidth).

You could make it even clearer, and save even more duplication, by turning this into a reusable component, that would encapsulate all the specific values etc, and then the prop names would take the part of the comment, e.g.:
```
<ResponsivePicture
  desktopJpgSrcset="/img/landing_page/step1_desktop1x.jpg 1x, /img/landing_page/step1_desktop2x.jpg 2x",
  desktopWebpSrcset="/img/landing_page/step1_desktop1x.webp 1x, /img/landing_page/step1_desktop2x.webp 2x",
  mobileJpgSrcset="..."
  mobileWebpSrcset="..."
  defaultImgSrc="..."
  className="..."
  alt="..."
/>
```

And if you wanna take it really far and eliminate all duplication, you could even do what you did with the other components - create a convention for the file names - and make this as simple as:
```
<ResponsivePicture
  imageBasePath="/img/landing_page/step1"
  className="..."
  alt="..."
/>
```
(This would also guarantee that any changes you want to make in the future, like change the logic of how image versions are selected as discussed in the comment above, would require changing only a single file. You could even use it in the testimonial and partner pictures, for maximum effect.)

None of this is necessary, of course, but even the minimal change of using a variable would've achieved the same level of readability, without the risk of getting out of sync with the code as comments often do, would've eliminated some duplication, and likely wouldn't have taken any longer to add than the comment did.

Reply:
Took out comments and did a little cleanup by setting up const for desktop pixel size